ChkdraftLib/Windows/MainWindows/GuiMap.cpp

```diff
@@ -17,7 +17,7 @@ GuiMap::GuiMap(Clipboard & clipboard, const std::string & filePath)
     bitmapHeight(0), bitmapWidth(0), currLayer(Layer::Terrain), currPlayer(0), zoom(1), RedrawMiniMap(true), RedrawMap(true),
     dragging(false), snapLocations(true), locSnapTileOverGrid(true), lockAnywhere(true),
     snapUnits(true), stackUnits(false), mapId(0), unsavedChanges(false), changeLock(false), undos(*this),
-    minSecondsBetweenBackups(1800), lastBackupTime(-1)
+    minSecondsBetweenBackups(1800), lastBackupTime(-1), panStartX(-1), panStartY(-1), panCurrentX(0), panCurrentY(0), panTimerID(0)
```

Comment:
Default values can be set in the header file, would make this file more readable and reduce duplication. Probably something for a refactor.

Reply:
indeed, default member initializers are generally better & I favor them in new code